### PR TITLE
Jsondb

### PR DESCRIPTION
**Pull request by Kviick**

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
         <module name="StoreIT" />
+        <module name="StorIT" />
```
Comment (steget) [inline .idea/compiler.xml]: Wrong name

File: src/main/java/storagesystem/model/Location.java [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 import javafx.scene.image.Image;
 
+/**
+ * Location tells us information about a place where a part of a team's inventory is.
```
Comment (steget) [inline src/main/java/storagesystem/model/Location.java]: An instance doesn't necessarily have to be for a part of a team's inventory.

```suggestion
 * Location tells us information about a place.
```

File: src/main/java/storagesystem/model/Location.java [hunk 3]
```diff
@@ -14,6 +20,11 @@ public Location(String name, String description, Image image) {
         this.image = image;
     }
 
+    public Location(){
+        this.name = "Hasen";
+        this.description = "Open the door";
```
Comment (steget) [inline src/main/java/storagesystem/model/Location.java]: Should be sent in as parameters instead.

File: src/main/java/storagesystem/services/GSONHandler.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    public static void addListToJson(List listToAdd, String fileName) throws IOException {
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        JsonArray oldJsonContent = gson.fromJson(new FileReader(fileName), JsonArray.class);
+        Writer writer = new FileWriter(fileName);
+        if (oldJsonContent == null) {
+            oldJsonContent = new JsonArray();
+        }
+        JsonArray newJsonContent = gson.toJsonTree(listToAdd, List.class).getAsJsonArray();
+        oldJsonContent.addAll(newJsonContent);
+        newJsonContent = oldJsonContent;
```
Comment (steget) [inline src/main/java/storagesystem/services/GSONHandler.java]: Switch places of these rows? Or just rename the object to jsonContent and use same throughout the entire method.

File: .idea/compiler.xml (hunk 1 above)
Comment (Kviick) [reply .idea/compiler.xml]: yep yep, fixed.

File: src/main/java/storagesystem/services/GSONHandler.java (hunk 4 above)
Comment (Kviick) [reply src/main/java/storagesystem/services/GSONHandler.java]: Nope. If switching rows the line oldJsonContent.addAll(newJsonContent); will be redundant. 
newJsonContent = oldJsonContent; is just for readability.

Comment (Kviick) [reply src/main/java/storagesystem/services/GSONHandler.java]: possibly change oldJsonContent to jsonContent and skip "newJsonContent = oldJsonContent;"

File: src/main/java/storagesystem/model/Location.java (hunk 3 above)
Comment (Kviick) [reply src/main/java/storagesystem/model/Location.java]: fixed in new commit

**Pull request merged**